contract-message-predicate/src/contract_message_predicate.sw

```diff
@@ -0,0 +1,82 @@
+predicate;
+
+dep transaction_utils;
+
+use std::{assert::assert, constants::ZERO_B256};
```

Comment:
Unsure of which version this is using however the [prelude](https://fuellabs.github.io/sway/v0.24.5/introduction/standard_library.html#standard-library-prelude) imports a bunch of things for you so that you do not have to.

Reply:
It looks like `assert` is now included, but not the constants